src/GitGlyph/GitBlameVisitor.cs

```diff
@@ -23,7 +23,7 @@ public GitBlameVisitor(Repository r, ILog logger)
         public string Visit(string text, IReadOnlyDictionary<string, string> context)
         {
             var path = System.IO.Path.GetFullPath(context[ContextKeys.FilePath]);
-            var blame = GetBlame(path)?.FirstOrDefault(bh => bh.FinalStartLineNumber.ToString() == context[ContextKeys.LineNumber]);
+            var blame = GetBlame(path)?.FirstOrDefault(bh => bh.FinalStartLineNumber.ToString() == (Int32.Parse(context[ContextKeys.LineNumber]) - 1).ToString());

```

Comment:
bh.FinalStartLineNumber returns an int, so there is no more need to convert to a string before comparing.